src/api/schemas/item.schema.ts

```diff
@@ -5,7 +5,8 @@ export const itemSchema = T.Object({
   id: T.String({ format: 'uuid' }),
   name: T.String(),
   description: T.String(),
-  image: T.String()
+  image: T.String(),
+  categoryId: T.String({ format: 'uuid' })
```

Comment:
This is wrong, you should shouw categories array in here, the data base allows for an item to have more than one category, so we should account for that when creating modifiying or showing items.